<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>INScoreLine</class>
 <widget class="QWidget" name="INScoreLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>132</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Object Name :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameLine">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>name</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>INScore Path :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="pathLine">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>500</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>/ITL/scene/</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="parametersBox">
         <property name="minimumSize">
          <size>
           <width>460</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>parameters</string>
         </property>
         <widget class="QCheckBox" name="display">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>71</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>display</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="size">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>size</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="transformation">
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>30</y>
            <width>115</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>transformation</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="color">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>30</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>color</string>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="dcolor">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>50</y>
            <width>71</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>dcolor</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="position">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>30</y>
            <width>87</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>position</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="dposition">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>50</y>
            <width>87</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>dposition</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="time">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>30</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>time</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="signal">
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>50</y>
            <width>87</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>signal</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="dtime">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>50</y>
            <width>71</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>dtime</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="toCreateButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>create object</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="typeList">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>curve</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ellipse</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>fastgraph</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>file</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>gmnf</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>graph</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>grid</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>html</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>layer</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>line xy</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>polygon</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>rect</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>svgf</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>txt</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="creationValue">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>init value</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="removeButton">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
